etc/encoding/opencast-movies.properties

```diff
@@ -48,7 +48,7 @@ profile.mux-av.work.name = mux audio and video
 profile.mux-av.work.input = stream
 profile.mux-av.work.output = visual
 profile.mux-av.work.suffix = -work.#{in.video.suffix}
-profile.mux-av.work.ffmpeg.command = -i #{in.video.path} -i #{in.audio.path} -shortest -c copy #{out.dir}/#{out.name}#{out.suffix}
+profile.mux-av.work.ffmpeg.command = -i #{in.video.path} -i #{in.audio.path} -shortest -c copy -map 0:v:0 -map 1:a:0 #{out.dir}/#{out.name}#{out.suffix}
```

Comment:
As this encoding profile is hard-coded in WOH prepare-av, you might want to introduce another encoding profile for WOH tracks-select. Or is this change supposed to be useful for WOH prepare-av, too?

Reply:
Ah yes. So when the prepare-av workflow receives a mediapackage with two tracks, and it's supposed to mux these together (so take the audio subtrack of track A and the video subtrack of track B), it actually strips the video subtrack from A and then merges the two streams together. This is unnecessary, since ffmpeg can mux the two streams. with a single command. I asked why this isn't being done in the profile above, and Greg replied that the profile might be because the code is ancient (meaning from times when ffmpeg wasn't able to mux as intelligently as it can, now).
To be extremely sure, I could introduce another profile.

Reply:
Thanks for the info! In that case, just modifying the existing encoding profile should be fine.